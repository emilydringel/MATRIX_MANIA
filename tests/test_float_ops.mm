def void testfloat(float a, float b){
    printf(a + b);
    printf(a - b);
    printf(a * b);
    printf(a / b);
    print(a == b);
    print(a == a);
    print(a != b);
    print(a != a);
    print(a > b);
    print(a >= b);
    print(a < b);
    print(a <= b);
}
def int main(){

    float c = 93.6;
    float d = 7.26;

    testfloat(c, d);

    testfloat(d, d);

}
