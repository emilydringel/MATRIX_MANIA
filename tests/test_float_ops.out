100.86
86.34
679.536
12.8926
0
1
1
0
1
1
0
0
14.52
0
52.7076
1
1
1
0
0
0
1
0
1